### ISSUE TITLE
A very general question

### ISSUE THREAD

**Issue by Steffen-W** [xorbit/GP2AP03VT00F_Arduino#1]
Hi @xorbit,

A very general question. I find it difficult to document the sensor. What are your experiences with the sensor? How much noise does it produce? I'm thinking about using it in my system. Thank you in any case for making the code available!

Regards Steffen-W

**Comment by xorbit** [comment]
Yes, Sharp is one of those companies who like to hide documentation and don't care unless you buy billions worth.  Very annoying.

Anyway, the readings seem quite stable and reproducible between parts.  The fine (close up) range varies smoothly, I'd say with mm resolution or so though my testing wasn't very thorough in that regard.  According to the spec the accuracy is +/- 6 mm, so you may need to use some calibration.

If you want to play with the part, I am now selling my extra breakout boards on Tindie if that would help in your evaluation: https://www.tindie.com/products/silicognition/sharp-gp2ap03vt00f-time-of-flight-breakout/

**Comment by Steffen-W** [comment]
Thank you for the answer. Sorry for the late response. For me, accuracy is not quite so important. The noise is actually much more relevant. The sensor has a very high resolution, which is much more interesting. Your circuit board is very interesting. Unfortunately, the shipping costs to Germany are very high.

**Comment by xorbit** [comment]
Yeah, not much I can do about that.  For other products I'd send you to Mouser to buy it locally, but this breakout is more of a one/off that will not make it to Mouser.

**Comment by TW-Openpilot** [comment]
I find this document, share for you
https://19697678.s21i.faiusr.com/61/ABUIABA9GAAgjsOMkQYo0cC__AM.pdf
